docs/source/getting-started/installation.rst

```diff
@@ -15,10 +15,18 @@ You need to install:
 2. JDK 8 or greater.
 
    You can get the JDK from `Zulu 8 JDK <https://www.azul.com/downloads/zulu/>`_ or `Oracle 8 JDK <http://www.oracle.com/technetwork/java/javase/downloads/index.html>`_ (requires you to accept Oracle's license).
+   As part of the installation process you might need to set up the ``JAVA_HOME`` variable. You can find here the instructions on how to do it on `Windows, Mac OS, and Linux <https://www.baeldung.com/java-home-on-windows-7-8-10-mac-os-x-linux>`_.
 
 2. Install the SDK
 *******************
 
+In order to install the DAML SDK you will first need to
+
+1. download it and
```

Comment:
Maybe swap out `download it` for `install it`? Neither running a curl command nor running the Windows installer is really summed up by “downloading”.